### DISCUSSION TITLE
Stabilization freezing objects balancing on their edge

### DISCUSSION THREAD

**Discussion by Zeblote** [NVIDIA-Omniverse/PhysX#138] [General]
Hi, I've noticed a fairly common problem after enabling the stabilization pass to stop piles of objects jittering on forever. If you place an object like a convex or a cube on its edge (or it happens to roll and loose enough momentum at just the right time while on an edge) it will often just freeze right there. Touching the object makes it slowly start rotating for roughly 0.2 seconds and then freeze again in a very unnatural position where users can tell that it's not right. Is there any solution to this?

**Comment by PierreTerdiman** [comment]
You can tweak the sleep & stabilization thresholds for each object. I suspect the default values might be a bit too aggressive here.
